### PR TITLE
Upload video added

### PR DESCRIPTION
**Pull request by ramtinak**

### CODE REVIEW

File: InstaSharper/Classes/VideoUploadJobResponse.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+namespace InstaSharper.Classes
+{
+    //{
```
Comment (a-legotin) [inline InstaSharper/Classes/VideoUploadJobResponse.cs:9]: remove commented code in every place in this PR

File: InstaSharper/API/Processors/MediaProcessor.cs [hunk 2]
```diff
@@ -115,6 +116,268 @@ internal class MediaProcessor : IMediaProcessor
...
+        }
+
+        public async Task<IResult<InstaMedia>> UploadVideoAsync(InstaVideo video, InstaImage imageThumbnail, string caption)
+        {
+            try
+            {
+                //POST /api/v1/upload/video/ HTTP/1.1
+                //Host: i.instagram.com
+                //upload_id=1458984764906&media_type=2&_uuid=ba913fe1-5c1a-4d66-b892-b77d16c3906a&_csrftoken=d6662d21cb381d03bd55ddddd6f76aa1
+                var instaUri = UriCreator.GetUploadVideoUri();
```
Comment (a-legotin) [inline InstaSharper/API/Processors/MediaProcessor.cs:128]: please make sure that your changes at least correctly build, before creating pull request:
```
error CS0117: 'UriCreator' does not contain a definition for 'GetUploadVideoUri'
```

**Review by a-legotin — CHANGES_REQUESTED** [review]
All checks have failed
+ couple of comments

**Comment by ramtinak** [comment]
Please remove this commit or at least close it.

**Pull request closed without merging**